demo/app/data/default-slide-data.ts

```diff
@@ -0,0 +1,97 @@
+type SlideTextElementName = 'text' | 'title' | 'title-caption';
+type SlideImgElementName = 'img';
+
+const IS_SELECTED = true;
+
+type Position = [x: number, y: number];
+
+export type SlideTextElement = {
+  elementType: SlideTextElementName;
+  text: string;
+  position: Position;
+  width?: number;
+  commentThreadId?: string;
+  lockedBy?: string;
+};
+
+export type SlideImgElement = {
+  elementType: SlideImgElementName;
+  src: string;
+  position: Position;
+  alt?: string;
+  caption?: string;
+  commentThreadId?: string;
+  lockedBy?: string;
+};
+
+export type SlideElement = SlideTextElement | SlideImgElement;
+
+// TODO: Define and render behaviour for elements that are `lockedBy` a user
+// TODO: Define and render behaviour for elements that have a `commentThreadId` started by a particular user
```

Comment:
Should we complete these TODOs before merging for Srushtika's purposes?

These TODOs refer to the placeholder UI elements that are displayed here:
<img width="661" alt="Screenshot 2023-03-30 at 09 57 46" src="https://user-images.githubusercontent.com/32373140/228785206-0dd51911-15d0-409e-a74d-37c20d98a8cc.png">
which should not be displayed in the slide Preview menu, but we will need some UI markers for Spatial Presence there too from the designs.

My perception is that, while we've completed a lot of the work for this demo placeholder, these UI elements are the bits that people will want to see as examples of what we have planned when Srushtika shows the demo page to them.

Reply:
Yes you are right, we should